README.md

```diff
@@ -1,372 +1,742 @@
+
+```python
+fig, ax = plt.subplots()
+```
+
+#### Rows and Columns
+
+We can also create multiple axes (i.e. multiple subplots) within a single figure by specifying the `nrows` and/or `ncols` arguments.
 
-Let's re-draw above plots in two different subplots. For this, you then pass the arguments (12x) - this tells you us that you have one row split into two columns.  You can replace x with 1 and 2 to address our subplots areas. You also pass `figsize =(x,y)` to `.figure()` function in order to define the size for our figure space (x and y values are in inches by default).
+For example, here we are creating 3 side-by-side axes within a figure:
 
 
 ```python
-# Define a new figure with matplotlib's .plot() function. Set the size of figure space
-new_figure = plt.figure(figsize=(10,4))
+fig, axes = plt.subplots(figsize=(11, 3), ncols=3)
+```
 
-# Add a subplot to the figure - a new axes
-ax = new_figure.add_subplot(121)
 
-# Add a second subplot to the figure - a new axes
-ax2 = new_figure.add_subplot(122)
+    
+![png](index_files/index_34_0.png)
+    
 
-# Generate a line plot on first axes
-ax.plot([1, 4, 6, 8], [10, 15, 27, 32], color='lightblue', linewidth=3, linestyle = '-.')
 
-# Draw a scatter plot on 2nd axes
-ax2.scatter([0.5, 2.2, 4.2, 6.5], [21, 19, 9, 26], color='red', marker='o')
+Here we have 3 axes stacked on top of one another:
 
-# Set the limits of x and y for first axes
-ax.set_xlim(0, 9), ax.set_ylim(5,35)
 
-# Set the limits of x and y for 2nd axes
-ax2.set_xlim(0, 9), ax2.set_ylim(5,35)
+```python
+fig, axes = plt.subplots(figsize=(3, 11), nrows=3)
+```
+
+
+    
+![png](index_files/index_36_0.png)
+    
+
 
-# Show the plot
-plt.show()
+#### Customizing Individual Axes
 
+If you try to apply methods like `set_facecolor` to this `axes` variable, you will get an error message:
 
+
+```python
+axes.set_facecolor("orange")
 ```
 
 
-![png](index_files/index_17_0.png)
+    ---------------------------------------------------------------------------
 
+    AttributeError                            Traceback (most recent call last)
 
-## More Subplots
+    <ipython-input-18-28b51014d60b> in <module>
+    ----> 1 axes.set_facecolor("orange")
+    
+
+    AttributeError: 'numpy.ndarray' object has no attribute 'set_facecolor'
+
+
+This is because `axes` is a collection of axes objects, not just one.
+
+We can access an individual axes object like this:
 
-In addition to adding subplots sequentially on the fly, as above, you can also predefine a grid of subplots like this:
 
 ```python
-fig, axes = plt.subplots(ncols=2, nrows=3) #2 columns, 3 rows
+fig, axes = plt.subplots(figsize=(11, 3), ncols=3)
+axes[1].set_facecolor("orange")
 ```
 
-From there, you can then plot on the individual subplots by accessing the subplot through the axes object:
+
+    
+![png](index_files/index_40_0.png)
+    
+
+
+Alternatively, we can "unpack" each axes object into its own variable like this:
```

Comment:
Building off the note above, re-writing this to be more explicit about `axes` being a list that we're unpacking:

Alternatively, instead of leaving each axes object in a list that we have to select each time we want to change something, we can "unpack" each axes object into its own uniquely-named variable like this:

Reply:
Added